apps/jackbnb-fe/src/components/PrimaryButton/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { Button } from "@chakra-ui/react";
+
+function PrimaryButton() {
```

Comment:
It should use const arrow function instead of function

Reply:
like:
```suggestion
const PrimaryButton: FC = () => {
```

note: FC is imported from react
```suggestion
import React, {FC} from "react";
```